lib/utils.pm

```diff
@@ -220,13 +220,13 @@ sub is_casp {
     return 1 unless $flavor;
 
     if ($flavor eq 'DVD') {
-        return get_var('FLAVOR') =~ /DVD/;    # Catch DVD and DVD-Staging
+        return get_var('FLAVOR') =~ /DVD/;    # DVD and Staging-?-DVD
     }
     elsif ($flavor eq 'VMX') {
-        return 1;                             # Everything other than DVD is VMX image
+        return get_var('FLAVOR') !~ /DVD/;    # If not DVD it's VMX
```

Comment:
yes, but how can there by anything else than VMX ? Also the change above.
I don't get the "duplicate" condition